core/src/main/java/com/cloud/storage/resource/StorageProcessor.java

```diff
@@ -33,6 +33,10 @@
 import com.cloud.agent.api.Answer;
 
 public interface StorageProcessor {
+
+    String REQUEST_TEMPLATE_RELOAD = "request template reload";
+    String COPY_NOT_NEEDED_FOR_DEPLOY_AS_IS = "copy volume not needed for deploy as is";
```

Comment:
@nvazquez where and how is this used?

Reply:
Used on the volume orchestrator answers from hypervisor to check if volume creation from template needs to be retried